code/src/components/Details.js

```diff
@@ -0,0 +1,59 @@
+import React, { useState, useEffect } from 'react';
+import { useParams, useNavigate } from 'react-router-dom';
+
+import { AiFillCaretLeft } from 'react-icons/ai';
+
+const Details = () => {
+	const { movieId } = useParams();
+
+	const [ details, setDetails ] = useState([]);
+
+	const navigate = useNavigate();
+
+	const onBackButtonClick = () => {
+		//Use this if go to previous page
+		//navigate(-1)
+
+		navigate('/');
+	};
+
+	useEffect(
+		() => {
+			fetch(`https://api.themoviedb.org/3/movie/${movieId}?api_key=25d003084b426a61d39038ca690f9a7c&language=en-US`).then((res) => res.json()).then((data) => setDetails(data));
+		},
+		[ movieId ]
+	);
+
+	console.log(details);
+
+	return (
```

Comment:
We noticed that your page not found component is not working. You can fix that by adding this to the line 28: `if (!details) {return <NotFound />}`